README.md

```diff
@@ -20,40 +20,48 @@ npm run build
 npm run lint
 ```
 
-## For the mobile app
+### Customize configuration
+See [Configuration Reference](https://cli.vuejs.org/config/).
+
+# Mobile app
 
 In order to be able to build the mobile app, you will need to follow these instructions first:
 [capacitor environment setup instructions](https://capacitorjs.com/docs/getting-started/environment-setup)
 
-### To generate the splash screens and the icons for mobile deployment
+## To generate the splash screens and the icons for mobile deployment
 
 If you need to do this, make sure you have cordova-res installed with `npm i -g cordova-res`
 
 Then put your icon and splash files in `resources/icon.png` and `resources/splash.png` respectively (see the files for minimum sizes) and then run:
 
 ```
+cd project root
 cordova-res android --skip-config --copy && cordova-res ios --skip-config --copy
 ```
 
-### To build the assets (the web app) for mobile deployment
+## Theme customization
+You can customize the app's look by setting your values in the `<project root>/public/config.json` file or by putting a separate scss variables declarations file in `<project root>/public/config.theme.scss` (see the `<project root>/src/assets/custom-variables.scss` for a reference of what you can customize). Note that if you set your variables in the `config.json` file, you **must** remove the `$` prepending the sass variables before setting them as json keys.
```

Comment:
This formulation was was not clear to me at the first read. And I'm no reference to make a really decisive suggestion, but here is my take : 
```
The app's theme color and properties can be customized by, either:
- Editing `<project root>/public/config.json` file thanks
  to a `theme` section as follow:

    {
      "lokapiHost": "example.org",
      ...
      "theme": {
        "color-1": "#E4F2F1",
        ...
        "btn-login-border-color": "#009688",
        "btn-login-background-color": "#009688"
      }
    }

The variables label in the "theme" section are the same than the
ones you'll find listed in `<project root>/src/assets/custom-variables.scss`.
Please be sure NOT to include the leading `$` in the `config.json` file.


- Providing a separate `<project root>/public/config.theme.scss`
  file following the `<project root>/src/assets/custom-variables.scss`
  format.
```
What do you think ?

Reply:
I updated the README file. Thanks for your suggestion.